src/main/kotlin/uk/gov/justice/digital/assessments/restclient/AssessmentUpdateRestClient.kt

```diff
@@ -57,38 +57,45 @@ class AssessmentUpdateRestClient {
       }
   }
 
+  data class OffenderContext (
```

Comment:
Noticed we would be passing multiple parts of the offender through here, so created an object to contain this information. Not quite sure where this should live though.. or this was the correct thing to do - open to suggestions 🤔

Reply:
Removed this code - as per above